bin/MadGraph5_aMCatNLO/cards/production/13TeV/MuMuJet_mll_0to60_LO/dyellell01234j_5f_LO_MLM_proc_card.dat

```diff
@@ -0,0 +1,20 @@
+import model sm-ckm_no_b_mass
+#switch to diagonal ckm matrix if needed for speed
+#import model sm-no_b_mass
+
+# define ell+ = e+ mu+ ta+
+# define ell- = e- mu- ta-
+
+define ell+ = mu+
+define ell- = mu- 
+
+#generate p p > ell+ ell- @0
+#add process p p > ell+ ell- j @1
+#add process p p > ell+ ell- j j @2
+#add process p p > ell+ ell- j j j @3
+#add process p p > ell+ ell- j j j j @4
+
+generate p p > ell+ ell- j @1
```

Comment:
Please remove unnecessary lines that are comment outed with #

Reply:
Done in the new commit